test/utils.spec.js

```diff
@@ -569,7 +567,7 @@ describe('ValidatyEvaluatorTests across Multiple Blocks', () => {
     validityEval.addResponseData(310, 'right_arrow', 1);
     validityEval.markAsCompleted();
     expect(testAddFlags).toHaveBeenLastCalledWith(
-      ['responseTimeTooSlow_DEL', 'accuracyTooLow_FSM', 'responseTimeTooFast_LSM'],
+      ['responseTimeTooFast_LSM'],
```

Comment:
What is the difference between this and line 605?

Reply:
570 represents a test where the response time is too fast only in the last sound matching (LSM) block. 605's test has the response time too fast in all 3 blocks.